board/pluto/S40network

```diff
@@ -75,8 +75,10 @@ create_system_files () {
 	fi
 
 	### /opt/config.txt ###
-	echo "# Device Configuration File"$'\r' > $CONF
-	echo "# Edit, Save and then Eject the USB Drive"$'\r' >> $CONF
+	echo "# (cat /sys/firmware/devicetree/base/model | tr / -"$'\r' > $CONF
```

Comment:
is `(cat /sys/firmware/devicetree/base/model`  supposed to run as a shell command ?
if so, is it missing a `$` sign ?
i could be missing something here though

Reply:
yeah, that is missing - sorry. Will fix.